test/integration/api/entities.js

```diff
@@ -2081,6 +2081,528 @@ describe('Entities API', () => {
 
   });
 
+  // Bulk API operations
+  describe('POST /datasets/:name/entities', () => {
+    // Tests that one would expect to find here are found above because this is additional
+    // functionality of an existing endpoint:
+    // - should return notfound if the dataset does not exist
+    // - should reject if the user cannot write
+    // - should reject creating new entity if dataset not yet published
+
+    it('should reject malformed json', testDataset(async (service) => {
+      const asAlice = await service.login('alice');
+
+      await asAlice.post('/v1/projects/1/datasets/people/entities')
+        .send([{ broken: 'json' }])
```

Comment:
Change to test something like `entities: [{ broken: 'json' }]`

Reply:
what if you dont pass an array to entities 
what if the data within one of these things is not an object